apps/web/src/pages/api/price/index.ts

```diff
@@ -0,0 +1,35 @@
+import { NextApiHandler } from 'next'
+
+export const config = {
+  runtime: 'nodejs',
+}
+
+const handler: NextApiHandler = async (req, res) => {
+  try {
+    const { currencyA, currencyB } = req.query
```

Comment:
Let's add cors plus token whitelist control. Just to prevent our endpoint from being abused

Reply:
Actually I'm thinking maybe we should make this api less general and more specific. Like `/api/prediction/price/arb/eth-usd` or `/api/prediction/usd-price/arb/eth`. What do you think?